lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -140,7 +140,8 @@ describe('PerfCategoryRenderer', () => {
     const diagnosticSection = categoryDOM.querySelectorAll('.lh-category > .lh-audit-group')[2];
 
     const diagnosticAudits = category.auditRefs.filter(audit => audit.group === 'diagnostics' &&
-        audit.result.score !== 1 && audit.result.scoreDisplayMode !== 'not-applicable');
+        audit.result.score < Util.PASS_THRESHOLD &&
```

Comment:
@paulirish was there a historical reason for this not just being `!Util.showAsPassed(audit.result)`?

Reply:
nope. that sounds great.

Reply:
:+1: Sorted in https://github.com/GoogleChrome/lighthouse/pull/5963/commits/6a1715a1afcf494f5caeca07eb67abfdd71e5aef.